template/macros.jinja

```diff
@@ -52,6 +52,15 @@
       tplroot ~ ':tofs:files_switch',
       default_files_switch
   ) %}
+  {#- Lookup files or fallback to source_files parameter #}
+  {%- if not lookup %}
+    src_files = source_files
+  {%- else %}
+    {%- set src_files = salt['config.get'](
+        tplroot ~ ':tofs:source_files:' ~ lookup,
+        source_files
+    ) %}
+  {%- endif %}
```

Comment:
This whole `if-else-endif` block is unnecessary, since the `config.get` will result in the default value of `source_files` being used.  If there's any discomfort with `lookup=None` as the top, this can be set to any other default value.  But it appears to work fine as it is.

Reply:
@myii thanks, I made some tests and you are right, the `salt['config.get']` is done with `template:tofs:source_files:None` which has no result so the second argument is used.

Reply:
Thanks.